minpy/core.py

```diff
@@ -130,16 +130,40 @@ class MinpyWrapperError(TypeError):
     pass
 
 
-def numpy_to_minpy(var):
+def _numpy_to_minpy(var):
     """ Convert a numpy array to minpy array """
     return array.Value.wrap(var)
 
 
-def minpy_to_numpy(var):
+def _minpy_to_numpy(var):
     """ Convert a minpy array to numpy array """
     return array.Value.wrap(var).get_data(ArrayType.NUMPY)
 
 
+def numpy_to_minpy(var):
+    """ Convert numpy array(s) to minpy array(s)
+
+    :param var: singular, list, or tuple of numpy array(s)
+    :return: singular, list, or tuple of minpy array(s)
+    """
+    if isinstance(var, tuple) or isinstance(var, list):
```

Comment:
For some reason, numpy arrays are also instances of collections.Iterable. Thus this applies a stricter constraint on input.